packages/apps/spaces/app/organization/[id]/src/components/Tabs/panels/AccountPanel/Contract/ContractBillingDetailsModal/EmailsInputGroup/EmailsInputGroup.tsx

```diff
@@ -100,50 +109,60 @@ export const EmailsInputGroup = ({
         </div>
       </div>
 
-      <Box width='100%'>
+      <Flex flexDir='column' flex={1} width='100%' gap={4}>
+        {(isFocused || !to) && (
+          <EmailSelect
+            formId={formId}
+            fieldName='billingEmail'
+            entryType='To'
+            placeholder='To email address'
+            autofocus={focusedItemIndex === 0}
+          />
+        )}
+
         {isFocused && (
           <>
-            <EmailSelect
-              formId={formId}
-              fieldName='billingEmail'
-              entryType='To'
-              autofocus={focusedItemIndex === 0}
-            />
             {(showCC || !!cc.length) && (
               <EmailSelect
                 formId={formId}
                 fieldName='billingEmailCC'
                 entryType='CC'
+                placeholder='CC email addresses'
                 autofocus={focusedItemIndex === 1}
               />
             )}
             {(showBCC || !!bcc.length) && (
               <EmailSelect
                 formId={formId}
                 fieldName='billingEmailBCC'
+                placeholder='BCC email addresses'
                 entryType='BCC'
                 autofocus={focusedItemIndex === 2}
               />
             )}
           </>
         )}
-      </Box>
+      </Flex>
 
       {!isFocused && (
-        <Flex flexDir='column' flex={1}>
-          <div
-            onClick={() => handleFocus(0)}
-            role='button'
-            aria-label='Click to input participant data'
-            className={cn('overflow-hidden', {
-              'flex-1': !bcc.length,
-            })}
-          >
-            <span className='text-sm font-semibold text-gray-700 mr-1'>To</span>
-            <Text color='gray.500' noOfLines={1}>
-              {to && <>{to.value ? to.value : `⚠️ [invalid email]`}</>}
-            </Text>
-          </div>
+        <Flex flexDir='column' flex={1} gap={4}>
```

Comment:
since you used only HTML tags can you replace the flex from chakra with <div className='flex flex-col flex-1 gap-4'>